engine/dlib/src/dmsdk/dlib/buffer.h

```diff
@@ -395,6 +398,10 @@ namespace dmBuffer
      * @return result [type:dmBuffer::Result] Returns BUFFER_OK if all went ok
      */
     Result UpdateContentVersion(HBuffer hbuffer);
+
+    Result SetBounds(HBuffer hbuffer, const dmVMath::Point3& min, const dmVMath::Point3& max);
+
+    Result GetBounds(HBuffer hbuffer, dmVMath::Point3& min, dmVMath::Point3& max);
```

Comment:
Now that we have the SetMetaDaty and GetMetaData, we don't need this Bounds on the dmBuffer.

We can simply do the `SetMetaData(buffer, dmHashString("AABB")...)` from the relevant places.
The meta data of hash("AABB") + 6 floats becomes the "API".

Reply:
Yeah, that was still fuzzy for me. Whether we do need Get/SetBounds or not.